HMMVoiceCreation.md

```diff
@@ -26,23 +26,115 @@ If you have already installed some of the required programs, please include thei
 
 This script generates a $MARY_BASE/lib/external/externalBinaries.config file that will be used by the Voice import tools to locate the necessary external programs.
 
-The necessary programs that this script checks are:  
-**HTS requirements:**  
-* [HTS-2.2_for_HTK-3.4.1.patch ](http://hts.sp.nitech.ac.jp/archives/2.2/HTS-2.2_for_HTK-3.4.1.tar.bz2)  
-* HTK-3.4.1 and HDecode patched with HTS-2.2_for_HTK-3.4.1.patch links:  
-* [HTK-3.4.1](http://htk.eng.cam.ac.uk/ftp/software/HTK-3.4.1.tar.gz) (you will need to register first)  
-* [HDecode](http://htk.eng.cam.ac.uk/prot-docs/hdecode.shtml) (you will need to register first)  
+The necessary programs that this script checks are:
+
+**Common tools**
+* awk (normally available in linux)
+* perl (normally available in linux)
+* bc (normally available in linux) 
+* tcl supporting snack, for example [ActiveTcl](http://www.activestate.com/Products/ActiveTcl/). Note that only ActiveTcl 8.4 includes snack; 8.5+ requires manual installation.  
```

Comment:
I think any reference to ActiveTcl should be removed -- they don't seem to supply a 64bit compatible snack library either...